### DISCUSSION TITLE
Audio files

### DISCUSSION THREAD

**Discussion by hardBSDk** [status-im/status-legacy#12624] [Ideas]
Implement a way to send audio files (playable inside of chat).

Code suggestion: https://github.com/DoubleSymmetry/react-native-track-player

![651c923864a7c92ab2](https://user-images.githubusercontent.com/55782009/127226764-b038b76b-1807-4b22-b888-969a73720581.jpeg)

![Telegram-new-media-player-2](https://user-images.githubusercontent.com/55782009/127226770-4a4c2eb8-1b72-4928-8df5-4df6a9dccf95.gif)

**Comment by du82** [comment]
This would be awesome, especially with the decentralized nature of Status.

Imagine a podcast that you run on the social feed and can listen to old episodes with an audio player.
